yarpl/include/yarpl/flowable/Subscribers.h

```diff
@@ -98,7 +113,13 @@ class Subscribers {
 
     void onError(folly::exception_wrapper error) override {
       Subscriber<T>::onError(error);
-      error_(std::move(error));
+      try {
+        error_(std::move(error));
+      } catch (const std::exception& exn) {
+        auto ew = folly::exception_wrapper{std::current_exception(), exn};
+        DLOG(FATAL) << "'error' method should not throw: " << ew.what();
+        LOG(ERROR) << "'error' method should not throw: " << ew.what();
```

Comment:
I think I don't need the LOG(ERROR)..
""DFATAL - logs FATAL error in debug mode (no NDEBUG macro defined)
  - reduces ERROR in production""